toolkit/scripts/tools.mk

```diff
@@ -13,6 +13,20 @@ $(call create_folder,$(BUILD_DIR)/tools)
 
 ######## GO TOOLS ########
 
+# The version as reported by 'go version'
+go_min_version = go1.19.0
```

Comment:
Let's take the Golang version from `toolkit/tools/go.mod` since that's our source of truth and this way we won't need to remember to update `tools.mk` every time we update the version of Golang.